pkg/web/static/openapi.json

```diff
@@ -1275,551 +1292,824 @@
           },
-          "reply_not_before": {
+          "expires_in": {
+            "type": "integer",
+            "description": "The duration in seconds until the access token expires."
+          }
+        }
+      },
+      "ReauthenticateRequest": {
+        "type": "object",
+        "description": "The request object for the reauthentication endpoint.",
+        "properties": {
+          "refresh_token": {
             "type": "string",
-            "format": "date-time"
+            "description": "The refresh token for reauthentication."
+          }
+        }
+      },
+      "PageQuery": {
+        "type": "object",
+        "description": "The request object for paginated endpoints.",
+        "properties": {
+          "page_size": {
+            "type": "integer",
+            "format": "int32",
+            "example": 50,
+            "default": 100,
+            "description": "The number of items to return per page."
           },
-          "extra_json": {
-            "type": "string"
+          "next_page_token": {
+            "type": "string",
+            "format": "base64",
+            "description": "The token for retrieving the next page of results."
           },
-          "transaction": {
-            "$ref": "#/components/schemas/Transaction"
+          "prev_page_token": {
+            "type": "string",
+            "format": "base64",
+            "description": "The token for retrieving the previous page of results."
           }
         }
       },
-      "SecureEnvelope": {
+      "IVMS101Certify": {
         "type": "object",
+        "description": "The request object for the IVMS101 certificate endpoint.",
         "properties": {
-          "id": {
+          "schema_type": {
             "type": "string",
-            "format": "ULID"
+            "description": "The type of the schema."
           },
-          "direction": {
+          "schema_version": {
             "type": "string",
-            "enum": [
-              "incoming",
-              "outgoing"
-            ]
+            "description": "The version of the schema."
           },
-          "envelope_id": {
+          "payload": {
+            "type": "object",
+            "description": "The payload of the IVMS101 certificate."
+          },
+          "public_key": {
             "type": "string",
-            "format": "UUID"
+            "description": "The public key of the IVMS101 certificate."
           },
-          "payload": {
+          "private_key": {
             "type": "string",
-            "format": "base64"
+            "description": "The private key of the IVMS101 certificate."
           },
-          "encryption_key": {
+          "public_key_password": {
             "type": "string",
-            "format": "base64"
+            "description": "The password for the public key of the IVMS101 certificate."
+          }
+        }
+      },
+      "Account": {
```

Comment:
The `account` object slightly differs from the object listed [here](https://github.com/trisacrypto/envoy/blob/main/pkg/web/api/v1/account.go#L21-L31). Would you mind taking another look?

I should note that we are aware that birthdate info should be added to a payload when sending a secure envelope and this will be included in a payload soon.